homeassistant/components/network/websocket.py

```diff
@@ -62,3 +69,28 @@ async def websocket_network_adapters_configure(
         msg["id"],
         {ATTR_CONFIGURED_ADAPTERS: network.configured_adapters},
     )
+
+
+@websocket_api.require_admin
+@websocket_api.websocket_command(
+    {
+        vol.Required("type"): "network/url",
```

Comment:
I wonder if we shouldn't just include this data in the existing `network` command? We can return adapters, configured adapters and the resolved URLs?

Reply:
I considered this but urls are handled in a different component in the frontend and it would make loading the data even more complicated